### DISCUSSION TITLE
Demo (video) of CalDAV Push with Nextcloud and DAVx⁵

### DISCUSSION THREAD

**Discussion by devvv4ever** [bitfireAT/webdav-push#25] [Announcements]
Hey people!

We're very happy to show you a first demo that we made during our investigations with WebDAV Push. It is only a first glimpse of whats coming to the WebDAV ecosystem in 2024 :dancers: But it shows where we're heading to!

If you're interested in more please contribute to our **discussions** here: 
https://github.com/bitfireAT/webdav-push/discussions

The **video** shows a "live" triggering of the sync in DAVx5 when adding an event in Nextcloud web calendar:
https://youtu.be/8ET6T6ZYp8k

The **current draft for our WebDAV Push**
https://github.com/bitfireAT/webdav-push/blob/main/webdav-push-draft.md

**Comment by petrarca-arezzo** [comment]
Very nice idea and this feature would be great! But please, don't go for Google - it's exactly what I would like to avoid (Google Play, FCM is not available on my phones anyhow)...

**Reply by rfc2822** [reply]
Why should we not make FCM available for people with Google services / FCM? Isn't it quite inconvenient if they'd had to install separate software? In our approach, open systems are as well supported by default as pre-installed FCM, if available.
